### PR TITLE
[New template ]: Add forum application to templates

### PR DESCRIPTION
**Pull request by Ekwuno**
This PR adds a template that uses Pages functions, Workers KV and durable Objects to build a commenting application.

### CODE REVIEW

**Review by lauragift21 — APPROVED** [review]
LGTM!

File: pages-example-forum-app/functions/api/code.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+		name: user.name,
+		username: user.login,
+		id: user.id,
+		avatar_url: user.avatar_url,
+	};
+}
+
+async function encodeJWT(user: any, secret: string) {
+	return jwt.sign(user, secret);
+	// return token; // Can't use JsonWebToken because it's not available in workers
```
Comment (eidam) [inline pages-example-forum-app/functions/api/code.ts:71]:
```suggestion
```

File: pages-example-forum-app/worker-durable/wrangler.toml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+compatibility_date = "2022-07-21"
+main = "src/index.ts"
+name = "worker-durable"
+
+[durable_objects]
+bindings = [
+  {name = "Learning_DurableObjects", class_name = "DurableObject"},
```
Comment (eidam) [inline pages-example-forum-app/worker-durable/wrangler.toml:7]: I would personally stick with a single naming (either camel or snake case)

**Review by eidam — COMMENTED** [review]
lgtm, just a couple of comments

File: pages-example-forum-app/worker-durable/wrangler.toml (hunk 2 above)
Comment (Ekwuno) [reply pages-example-forum-app/worker-durable/wrangler.toml:7]: Any other reason for this so I can be mindful in the future?

**Pull request merged**